src/app/auth/duck/index.ts

```diff
@@ -1,4 +1,4 @@
 import authReducer from './reducers';
 export { default as authOperations } from './operations';
-export { Creators as authActionCreators } from './actions';
+export { AuthActions as authActionCreators } from './actions';
```

Comment:
Don't think this requires a change, but you could just export this as `Actions`, because the import will likely look like:

```
import authDuck from './auth/duck'
...
authDuck.Actions.Creators.callFoo()
```

What you have works too though, because we also are doing a lot of importing the actions directly:

```
import { Actions as AuthActions } from './auth/duck/actions'
```
Your solution here lets us skip that import alias.